.github/workflows/deploy-main-docs.yml

```diff
@@ -0,0 +1,60 @@
+name: deploy-main-docs
```

Comment:
Do I undestand it correctly that the whole point of building the documentation in GitHub actions rather than in Vercel is that we can control *when* the documentation is build? I.e. only when files inside `docs` have changed and there isn't `[skip ci]` in commit message?

Reply:
Yes, i think is more interesting to control in the CI when the docs are deployed. Now Vercel is deploying the docs always also when the docs doesn't change, so probabbly this strategy it's better.

Reply:
Vercel should continue creating the temporal domains for pull requests, but for main/next has more sense to me, to control it on the ci. But as you want, both options are valid.